woodwork/logical_types.py

```diff
@@ -31,6 +31,19 @@ def __str__(self):
         return str(self.__class__)
 
 
+class Address(LogicalType):
+    """Represents Logical Types that contain address values.
+
+    Examples:
+        .. code-block:: python
+
+            ['1 Miller Drive, New York, NY 12345', '1 Berkeley Street, Boston, MA 67891']
+            ['26387 Russell Hill, Dallas, TX 34521', '54305 Oxford Street, Seattle, WA 95132']
+    """
+    primary_dtype = 'string'
+    standard_tags = set()
```

Comment:
Not a big deal, but you don't need to define this here - it will inherit from `LogicalType` since the default is an empty set.